src/api/database/models/map/Landmark.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  Column,
+  Table,
+  Model,
+  PrimaryKey,
+  ForeignKey,
+  Unique,
+  BelongsTo,
+} from "sequelize-typescript";
+import { UUID, STRING, JSONB, DATE } from "sequelize";
+import User from "../User";
+import ZodColumn from "../../modelHelpers/ZodColumn";
+import z from "zod";
+
+@Table
+class Landmark extends Model {
+  @PrimaryKey
+  @Column(STRING)
+  landmark: string;
+
+  @Unique
+  @PrimaryKey
+  @ForeignKey(() => User)
+  @Column(UUID)
```

Comment:
The primary key should be (userId, landmark, createdAt), in this order. createdAt is a built-in column in `Model`, so I believe you need to use class-level primary key definition instead of `@PrimaryKey` here.